Swift/Tests/URLEndpointListenerTest.swift

```diff
@@ -21,8 +21,8 @@ import XCTest
 
 @available(macOS 10.12, iOS 10.3, *)
 class URLEndpointListenerTest: ReplicatorTest {
-    let wsPort: UInt16 = 4984
-    let wssPort: UInt16 = 4985
+    let wsPort: UInt16 = 2
+    let wssPort: UInt16 = 3
```

Comment:
Port 2 and 3 are reserved. Is there a reason to change the port number here? If it is because that they are the same as default SG port number, you can change to something like 4084 and 4085.

Reply:
yes, that was exactly the reason I wanted to change. Will change to 4084 and 4085. 

I've also set the kConnRefusedTarget to port 1. Will change that to 4083 as well.